conf/supervisord-dev.conf

```diff
@@ -3,14 +3,14 @@ nodaemon=true
 silent=true
 
 [program:web]
-command=gunicorn --bind :8001 --reload --timeout 0 --paste conf/development.ini
+command=gunicorn --paste conf/development.ini --config conf/gunicorn-dev.conf.py
 stdout_events_enabled=true
 stderr_events_enabled=true
 stopsignal=KILL
 stopasgroup=true
 
 [program:web-https]
-command=gunicorn --bind :48001 --reload --timeout 0 --paste conf/development.ini --certfile=.certificates/localhost/localhost.crt --keyfile=.certificates/localhost/localhost.key
+command=gunicorn --paste conf/development.ini --config conf/gunicorn-dev.conf.py --bind 0.0.0.0:48001 --certfile=.certificates/localhost/localhost.crt --keyfile=.certificates/localhost/localhost.key
```

Comment:
We could have separate Gunicorn config files for `web` and `web-https` in dev. But I went for just using the same config file and then using command line args to override `--bind` and add `--certfile` and `--keyfile`